src/controllers/accomodation.controller.js

```diff
@@ -59,6 +59,17 @@ class Accommodation {
     };
     response.successMessage(res, 'accommodation data', 201, data);
   }
+
+  /**
+   * Book an accommodation facility
+   * @param {Object} req The request object
+   * @param {Object} res The response object
+   * @returns {Promise} res
+   */
+  static async bookAccommodation(req, res) {
+    const query = await accomodationServices.bookAccommodation(req.user.id, req.body.accommodationId, req.body.roomType, req.body.numberOfBeds, req.body.departureDate, req.body.checkoutDate);
```

Comment:
@VedaDom wouldn't be enough to just indicate the roomID of the booked room instead of passing these detailed data (i.e. roomType and numberOfBeds)? Remember that a user should book a particular room. You can first get an ID of the available room given it's type.